FastSimulation/Configuration/python/Reconstruction_BefMix_cff.py

```diff
@@ -13,7 +13,12 @@
 from RecoVertex.BeamSpotProducer.BeamSpot_cff import offlineBeamSpot
 
 # and of course it needs tracker hits
-from FastSimulation.TrackingRecHitProducer.SiTrackerGaussianSmearingRecHitConverter_cfi import siTrackerGaussianSmearingRecHits
+
+#uses the old RecHit producer
+#from FastSimulation.TrackingRecHitProducer.SiTrackerGaussianSmearingRecHitConverter_cfi import siTrackerGaussianSmearingRecHits
+#uses the new RecHit producer
+from FastSimulation.TrackingRecHitProducer.NewRecHitConverter_NoMerge_cfi import trackingRecHitProducerNoMerge as siTrackerGaussianSmearingRecHits
```

Comment:
use proper file names and module names
(no *New*RecHitConverter..., get rid of the name siTrackerGaussianSmearingRecHits)